extensions/amp-analytics/0.1/test-e2e/test-cbe.js

```diff
@@ -41,13 +41,11 @@ describes.endtoend(
 
       // Sleep 1 second for the blur event to be sent
       await sleep(1000);
-      await expect(
-        'https://foo.com/event?type=blur&eventId='
-      ).to.have.been.sent;
+      await expect('https://foo.com/event?type=blur&eventId=inputText2').to.have
+        .been.sent;
     });
 
     it('click on dropdown menu to focus and then blur', async () => {
-
       const focusedDropdown = await controller.findElement('#numChild');
       const blurredDropdown = await controller.findElement('#numChild2');
```

Comment:
nit: can you rename these to something more descriptive ex. `firstChild`, `secondChild`?

Reply:
Should be resolved with the previous commit.